views/openacademy_menu.xml

```diff
@@ -25,7 +33,17 @@
 
     <menuitem id="menu_session"
         name="Sessions"
-        parent="menu_root"
+        parent="menu_root"/>
+
+    <menuitem id="sub_menu_session"
+        name="Sessions"
```

Comment:
This is the same name as above

Reply:
Sorry, I just noticed one menu was inside the other, so both having the same name is expected.

Reply:
This is still missing